plugins/DuskVerb/tools/sweep_decay_vvp.py

```
@@ -0,0 +1,106 @@
+import numpy as np
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, HERE)
+from perceptual_diff import (
+    load_mono, c80, a_weighted_rms_db,
+    bass_ratio, treble_ratio, spectral_crest_db,
+    edt, box_ratio_db, k_weighted_lufs,
+)
+
+RENDER_BIN = '/home/marc/projects/plugins/build/tests/duskverb_render/duskverb_render'
+LEX_IMP = '/tmp/lex_compare/lex_v3_impulse.wav'
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/lex_compare/lex_v3_impulse.wav"

(S108)